<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TunnelGeneratorWindow</class>
 <widget class="QMainWindow" name="TunnelGeneratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TunnelGenerator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>571</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButtonTS">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>94</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TrackSpace</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFTS">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>108</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>FullTunnelSpace</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonTC">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>114</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Trackbed Concrete</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonTL">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Traffic Light</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Level of Detail 4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>94</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonW">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Walkway</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Level of Detail 5</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFC">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>170</y>
       <width>94</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Floor Concrete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Level of Detail 3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonLS">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>94</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>LiningSpace</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFS">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>94</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>FloorSpace</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonS">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>170</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Segment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave_ifc4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>94</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Ifc4 file</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonAGS">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>108</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AnnularGapSpace</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonIS">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>93</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>InteriorSpace</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Level of Detail 2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonCS">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>108</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ClearanceSpace</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSS">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>93</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ServiceSpace</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>305</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave_tunnel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Ifc4_Tunnel file</string>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_14">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>140</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_16">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>31</width>
       <height>12</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>571</width>
      <height>171</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;With this device you can combine various obj-files and convert them to one ifc-file.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This is the procedure:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;1.&lt;/span&gt; select a tunnel-element-species via&lt;span style=&quot; font-weight:600;&quot;&gt; radio-buttons&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;2.&lt;/span&gt; open the fitting obj-file via &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;Open File&amp;quot;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;3.&lt;/span&gt; you can &lt;span style=&quot; font-weight:600;&quot;&gt;repeat step 3&lt;/span&gt; &lt;span style=&quot; font-weight:600;&quot;&gt;and step 4 &lt;/span&gt;as often as you wish&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;4.&lt;/span&gt; click &lt;span style=&quot; font-weight:600;&quot;&gt;either&lt;/span&gt; &amp;quot;Generate &lt;span style=&quot; font-weight:600;&quot;&gt;Ifc4&lt;/span&gt;-file&amp;quot;&lt;span style=&quot; font-weight:600;&quot;&gt; or &lt;/span&gt;&amp;quot;Generate&lt;span style=&quot; font-weight:600;&quot;&gt; Ifc4_tunnel&lt;/span&gt;-file&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Wait a &lt;span style=&quot; font-weight:600;&quot;&gt;few minutes&lt;/span&gt; after step 5 until you see the window &amp;quot;File has been generated&amp;quot;. Besides do&lt;span style=&quot; font-weight:600;&quot;&gt; not combine two times the same obj-file&lt;/span&gt;. At last be aware, that the geometrical-model-space of every element of an LoD has to be &lt;span style=&quot; font-weight:600;&quot;&gt;free of points from other elements of the same LoD&lt;/span&gt;. At last be aware that you just can insert a tunnel-element-species if you have inserted the element that is higher in hierarchy, too.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>280</y>
      <width>345</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>31</width>
      <height>12</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
